src/configuration-parser.ts

```diff
@@ -19,6 +19,8 @@ export class ConfigurationParser {
             pending: "* ",
             successful: ConfigurationParser.isWindows ? "\u221A " : "✓ ",
         },
+        // tslint:disable-next-line:no-unbound-method
+        print: console.log,
```

Comment:
Any reason to reference directly `console.log`?
By using a wrapper we could spare us the console.log mock